foundry/external_data_architectures.py

```diff
@@ -1,6 +1,7 @@
 import numpy as np
 import torch
 from torch.utils.data import Dataset
+from tensorflow.keras.utils import Sequence
 
 class FoundryDataset_Torch(Dataset):
```

Comment:
Could you add a short statement about what these classes are for. Just something basic like "Dataset that automatically loads from a Foundry-style resource."